### PR TITLE
Reinstall guest agent when existing one is masked

### PR DESCRIPTION
**Pull request by kylazhang**

### CODE REVIEW

**Comment by kylazhang** [comment]
Any one please help to review it asap as it affect out qe consumption test, thanks

File: virttest/libvirt_vm.py [hunk 1]
```diff
@@ -2393,6 +2393,11 @@ def _start_ga():
             if not _is_ga_running():
                 cmd = "service qemu-guest-agent start"
                 status, output = session.cmd_status_output(cmd)
+                # Sometimes agent serverice may be masked so need to reinstall
```
Comment (PandaWei) [inline virttest/libvirt_vm.py]: Please fix typo

**Comment by PandaWei** [comment]
Is there any failed case caused by the problem?

**Comment by PandaWei** [comment]
Please attach the details, such as error message..

**Comment by kylazhang** [comment]
@PandaWei Sometimes it may report error

  ERROR| VMError: Start qemu-guest-agent failed:
  ERROR| Redirecting to /bin/systemctl start  qemu-guest-agent.service
  ERROR| Failed to issue method call: Unit qemu-guest-agent.service is masked.

The resolve method is to reinstall the package

File: virttest/libvirt_vm.py (hunk 1 above)
Comment (kylazhang) [inline virttest/libvirt_vm.py]: ok. thanks

**Comment by PandaWei** [comment]
Thanks for you reply.
But fix the masked problem by reinstall is a bad idea，
which is time-consuming operation and unnecessary.
Please consider using 'systemctl unmask  service'  to fix it.
I‘m not sure, please have a try at least.

**Comment by kylazhang** [comment]
@PandaWei Thanks for your advice, I try and it does not work. Any other methods?

**Comment by PandaWei** [comment]
I have no suggestion so for.
Let's listen to the view of others.

**Comment by will-Do** [comment]
This should be BZ#https://bugzilla.redhat.com/show_bug.cgi?id=1188148 , as file qemu-guest-agent.service is broken(link to /dev/null or file corrupted), then you can rebuild the service link or reinstall it.
I'm okay with this patch, but you can add a more precise desc in the code

**Comment by kylazhang** [comment]
The desc has been changed. Thanks

**Comment by will-Do** [comment]
Thx

**Pull request merged**